src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write your code here
+function cloneColumn(section) {
+  Array.from(section.rows).forEach((row) => {
+    const secondCell = row.cells[1];
+    const lastCell = row.cells[row.cells.length - 1];
+
+    if (secondCell && lastCell) {
+      const clonedCell = secondCell.cloneNode(true);
+
+      row.insertBefore(clonedCell, lastCell);
+    }
+  });
+}
+
+const thead = document.querySelector('thead');
```

Comment:
Consider adding error handling for cases where the elements are not found. Right now, if any of these elements are not found, it will throw an error when you try to use them in `cloneColumn` function.